ios/scripts/generate-error-messages.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh -l
+
+# This ensures that relative paths are correct no matter where the script is executed
+cd "$(dirname "$0")"
+
+cd ../..
+
+echo "Generating Localizable files for error messages in specified languages"
+./gradlew generateErrorsTwine
+
```

Comment:
Mentioning this here as well. This will have a problem if twine contains some strings with quotes.


https://github.com/icerockdev/moko-resources/issues/462

Reply:
The simplest solution is probably replacing the escaped quotes with sed before running the moko gradle task:

`sed -i '.bak' -e 's@\\"@"@g' "$STRINGS_FILE"`

Reply:
Thanks, I will borrow your solution from Stravenky 🙂